src/config/routes.php

```diff
@@ -1,3 +1,4 @@
 <?php
 
 $router->get("/" , "HomeController::index");
+$router->get("/category/{id}/products" , "HomeController::show");
```

Comment:
Using `HomeController` to display product lists is mixing concerns. Let's separate the concerns and create a new controller. You can route to `CategoryController::show` or `ProductController::index`.